DESCRIPTION

```diff
@@ -26,17 +26,12 @@ Imports:
 Suggests:
     blockCV,
     cowplot,
-    knitr,
-    lgr,
     sf,
     stringr,
     testthat,
     vdiffr
-VignetteBuilder:
-    knitr
 Remotes:
     be-marc/blockCV
-ByteCompile: true
```

Comment:
Ah yes, ByteCompile now defaults to true, doesn't it?

Reply:
Yep